src/util/table/EnhancedTableHead.js

```diff
@@ -50,9 +50,7 @@ class EnhancedTableHead extends React.Component {
                                 indeterminate={isInDeterminate}
                                 checked={numSelected === rowsInPage}
                                 onChange={onSelectAllClick}
-                                inputProps={{
-                                    "aria-label": getMessage("selectAll"),
-                                }}
+                                inputProps={selectAllLabel}
```

Comment:
I was figuring this would be `inputProps={{"aria-label": selectAllLabel}}` -- otherwise the prop is sorta misnamed, I think, since it's really a whole object of input props.

Reply:
Oops. Good Catch!.